packages/@react-spectrum/avatar/src/Avatar.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {classNames, dimensionValue, useStyleProps} from '@react-spectrum/utils';
+import {FocusRing} from '@react-aria/focus';
+import React, {useRef} from 'react';
+import {SpectrumAvatarProps} from '@react-types/avatar';
+import styles from '@adobe/spectrum-css-temp/components/avatar/vars.css';
+import {useAvatar} from '@react-aria/avatar';
+import {useProviderProps} from '@react-spectrum/provider';
+
+/**
+ * An avatar is a thumbnail representation of an entity, such as a user or an organization.
+ */
+export function Avatar(props: SpectrumAvatarProps) {
+  const {
+    isDisabled,
+    size = 'avatar-size-100',
+    src,
+    ...otherProps
+  } = useProviderProps(props);
+
+  const ref = useRef();
+
+  const {avatarProps} = useAvatar(props, ref);
+  const {alt, ...otherAvatarProps} = avatarProps;
```

Comment:
Does `alt` need to be separated here? Can't you just spread `avatarProps` on img?

Reply:
that makes me think that avatarProps doesn't have the right return type, it should probably specify that it always returns the alt even if it's just a pass through